proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,219 @@
+# AWS CLI Contributing Guide
+
+Proposal         | Metadata
+---------------- | -------------
+**Author**       | Kenneth Daily
+**Status**       | Proposed
+**Created**      | 19-October-2021
+
+## Abstract
+
+This document proposes standards and processes to improve and publish community
+expectations for contributions. It defines the process for proposing,
+submitting, and reviewing a contribution. This proposal will make it easier for
+the community to successfully contribute to the AWS CLI.
+
+## Motivation
```

Comment:
Seven goals is a lot. Match them up with the examples and simplify.

Reply:
Main goals:

1. Reduce open PRs for what we'll work on and what's available.
2. Need prioritization, based on what community wants, we can focus finite bandwidth to see them end-to-end.
3. Be explicit with what's available to work on.
4. No PR goes unresponded.